packages/manager/src/mocks/serverHandlers.ts

```diff
@@ -2325,6 +2325,40 @@ export const handlers = [
 
     return HttpResponse.json(response);
   }),
+  http.get('*/v4/monitor/services', () => {
+    const response = {
+      data: [
+        {
+          label: 'Linodes',
+          value: 'linode',
+        },
+        { label: 'DbaaS', value: 'dbaas' },
+      ],
+    };
+    return HttpResponse.json(response);
+  }),
+  http.post('*/monitor/alerts', async ({ request }) => {
+    const reqBody = await request.json();
+    const response = {
```

Comment:
Please define factories instead of hardcoding

Reply:
We still don't have some backend process planned out with respect to fetching notifications from the sink_ids as of now. So apart from that only the alert_id's are hardcoded. Even for this current state the expectations are to write factories for the mock responses ?